### PR TITLE
fix(almin): tryUpdateState should be called before actual finished

### PR DESCRIPTION
**Pull request by azu**
from #241 

This PR fixes to update on completed payload.

Before:

- When call tryUpdateState on Completed, the useCase is already finished.

After:

- When call tryUpdateState on Completed, the useCase is not finished.
- After tryUpdateState is finished, the useCase is  actually finished.

### CODE REVIEW

File: packages/almin/test/StoreGroup-edge-case-test.js [hunk 1]
```diff
@@ -2,14 +2,44 @@
...
-import { StoreGroup } from "../src/UILayer/StoreGroup";
-import { NoDispatchUseCase } from "./use-case/NoDispatchUseCase";
-import ReturnPromiseUseCase from "./use-case/ReturnPromiseUseCase";
+import { Context, Dispatcher, Store, StoreGroup, UseCase } from "../src/";
+import { createUpdatableStoreWithUseCase } from "./helper/create-update-store-usecase";
+import { AsyncUseCase } from "./use-case/AsyncUseCase";
+import { SyncNoDispatchUseCase } from "./use-case/SyncNoDispatchUseCase";
 
 describe("StoreGroup edge case", function() {
+    describe("when {A,B}Store#emitChange on UseCase is completed", () => {
```
Comment (azu) [inline packages/almin/test/StoreGroup-edge-case-test.js]: That is test case for this issue.
